molgenis-data-rest/src/test/java/org/molgenis/data/rest/RestControllerIT.java

```diff
@@ -155,7 +155,7 @@ public void deleteWithoutWritePermissionFails()
 			   .then()
 			   .statusCode(UNAUTHORIZED)
 			   .body("errors.message[0]",
-					   equalTo("No [WRITE] permission on entity type [User authority] with id [sys_sec_UserAuthority]"));
+					   equalTo("No [READ] permission on entity type [User authority] with id [sys_sec_UserAuthority]"));
```

Comment:
Shouldn't this just stay WRITE? Since the function is named: deleteWithoutWritePermissionFails

Reply:
In order to be able to CREATE/UPDATE/DELETE 'User authority' rows, you require READ access on the metadata.